bundle/render_templates.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+set -x
+
+env
+
+csv_name="cert-manager-operator.clusterserviceversion.yaml"
+csv_file="${MANIFESTS_DIR}/${csv_name}"
+if [ ! -f $csv_file ]; then
+   echo "CSV file not found in the upstream!"
+   exit 5
+fi
+
+yq e -i ".metadata.annotations.createdAt=\"$(date -u +'%Y-%m-%dT%H:%M:%S')\"" "${csv_file}"
+yq -i 'del(.spec.relatedImages)' "${csv_file}"
+
+sed -i "s/quay.io\/jetstack\/cert-manager-webhook.*/registry-proxy.engineering.redhat.com\/rh-osbs\/cert-manager-jetstack-cert-manager-rhel9:${IMAGE_TAG}/g" ${csv_file}
+sed -i "s/quay.io\/jetstack\/cert-manager-controller.*/registry-proxy.engineering.redhat.com\/rh-osbs\/cert-manager-jetstack-cert-manager-rhel9:${IMAGE_TAG}/g" ${csv_file}
+sed -i "s/quay.io\/jetstack\/cert-manager-cainjector.*/registry-proxy.engineering.redhat.com\/rh-osbs\/cert-manager-jetstack-cert-manager-rhel9:${IMAGE_TAG}/g" ${csv_file}
+sed -i "s/quay.io\/jetstack\/cert-manager-acmesolver.*/registry-proxy.engineering.redhat.com\/rh-osbs\/cert-manager-jetstack-cert-manager-acmesolver-rhel9:${IMAGE_TAG}/g" ${csv_file}
```

Comment:
`s/gcr.io\/kubebuilder\/kube-rbac-proxy.*/registry.redhat.io\/openshift4\/ose-kube-rbac-proxy:${KUBE_RBAC_PROXY_VERSION}/g` seems like a nice example motivation to me, because it's already using `registry.redhat.io` which will be prod customer facing.

Reply:
I will explore how we can achieve this using renovators.